internal/command/container.go

```diff
@@ -0,0 +1,393 @@
+package command
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/fyne-io/fyne-cross/internal/icon"
+	"github.com/fyne-io/fyne-cross/internal/log"
+	"github.com/fyne-io/fyne-cross/internal/volume"
+	"golang.org/x/sys/execabs"
+)
+
+const (
+	// fyneBin is the path of the fyne binary into the docker image
+	fyneBin = "/usr/local/bin/fyne"
+	// gowindresBin is the path of the gowindres binary into the docker image
+	gowindresBin = "/usr/local/bin/gowindres"
+)
+
+type ContainerRunner interface {
+	NewImageContainer(arch Architecture, OS string, image string) ContainerImage
+
+	Debug(v ...interface{})
+	GetDebug() bool
+}
+
+type AllContainerRunner struct {
+	ContainerRunner
+
+	Env  map[string]string // Env is the list of custom env variable to set. Specified as "KEY=VALUE"
+	Tags []string          // Tags defines the tags to use
+
+	vol volume.Volume
+
+	debug bool
+}
+
+type ContainerImage interface {
+	Cmd(vol volume.Volume, opts Options, cmdArgs []string) *execabs.Cmd
+	Run(vol volume.Volume, opts Options, cmdArgs []string) error
+	Prepare() error
+	Finalize(srcFile string, packageName string) error
+
+	GetArchitecture() Architecture
+	GetOS() string
+	GetID() string
+	GetTarget() string
+	GetEnv(string) (string, bool)
+	SetEnv(string, string)
+	SetMount(string, string)
+	AppendTag(string)
+
+	GetRunner() ContainerRunner
+}
+
+type ContainerMountPoint struct {
+	LocalHost   string
+	InContainer string
+}
+
+type AllContainerImage struct {
```

Comment:
Not sure about the naming here. The `All` prefix makes me to think about a collection